flyteidl/protos/flyteidl/plugins/kubeflow/tensorflow.proto

```diff
@@ -18,10 +18,13 @@ message DistributedTensorflowTrainingTask {
   // Chief replicas spec
   DistributedTensorflowTrainingReplicaSpec chief_replicas = 3;
 
+  // Evaluator replicas spec
+  DistributedTensorflowTrainingReplicaSpec evaluator_replicas = 4;
+
   // RunPolicy encapsulates various runtime policies of the distributed training
   // job, for example how to clean up resources and how long the job can stay
   // active.
-  RunPolicy run_policy = 4;
+  RunPolicy run_policy = 5;
```

Comment:
We should not change the type of an existing field in a protobuf message as per https://protobuf.dev/programming-guides/dos-donts/ as that breaks backwards compatibility.

Reply:
Thanks really much!
This is pretty useful information, thanks a lot for your time.
I will improve it.